### PR TITLE
AC Stub Test - LRAC-8343 | master 

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-8343

### CODE REVIEW

**Comment by alexcandidos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase [hunk 1]
```diff
@@ -36,6 +36,33 @@ definition {
...
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8340 | Test Summary: 180 Days time period filter in Page table"
+	@ignore = "true"
+	@priority = "3"
+	test PeriodFilter180DaysPageTable {
+
+		// TODO LRAC-8340 ShowsListOfPages pending implementation
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:54]: hey @LucasAcacio , please make sure the test on the TODO comment is the same of the test ;)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase [hunk 2]
```diff
@@ -36,6 +36,33 @@ definition {
...
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8341 | Test Summary: Last Year time period filter in Page table"
+	@ignore = "true"
+	@priority = "3"
+	test PeriodFilterLastYearPageTable {
+
+		// TODO LRAC-8341 ShowsListOfPages pending implementation
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:63]: hey @LucasAcacio , please make sure the test on the TODO comment is the same of the test ;)

**Review by alexcandidos — CHANGES_REQUESTED** [review]
Hi @LucasAcacio , please take a look at my comments.

**Comment by LucasAcacio** [comment]
Required changes done

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**